dlp/api/DlpSample/InspectTemplates.cs

```diff
@@ -0,0 +1,117 @@
+﻿using Google.Api.Gax;
+using Google.Cloud.Dlp.V2;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace GoogleCloudSamples
+{
+    class InspectTemplates
+    {
+        // [START dlp_create_inspect_template]
+        public static string CreateInspectTemplate(
+            string projectId, 
+            string displayName,
+            string description,
+            int likelihood, 
+            int maxFindings,
+            bool includeQuote,
+            string infoTypes)
+        {
+            DlpServiceClient client = DlpServiceClient.Create();
+
+            var request = new CreateInspectTemplateRequest
+            {
+                Parent = $"projects/{projectId}",
+                InspectTemplate = new InspectTemplate
+                {
+                    DisplayName = displayName,
+                    Description = description,
+                    InspectConfig = new InspectConfig
+                    {
+                        MinLikelihood = (Likelihood)likelihood,
+                        Limits = new InspectConfig.Types.FindingLimits
+                        {
+                            MaxFindingsPerRequest = maxFindings
+                        },
+                        IncludeQuote = includeQuote
+                    },
+                }
+            };
+
+            var response = client.CreateInspectTemplate(request);
+
+            Console.WriteLine("Inspect template details:");
+            Console.Write($"name: {response.Name}, createTime: {response.CreateTime}");
+
+            return response.Name;
+        }
+        // [END dlp_create_inspect_template]
+
+        // [START dlp_delete_inspect_template]
+        public static object DeleteInspectTemplate(string projectId, string templateName)
+        {
+            DlpServiceClient client = DlpServiceClient.Create();
+
+            var request = new DeleteInspectTemplateRequest
+            {
+                Name = templateName
+            };
+
+            client.DeleteInspectTemplate(request);
+            Console.WriteLine($"Template {templateName} was deleted");
```

Comment:
Do we want to verify anything about the response before claiming the template was deleted? Or does the DeleteInspectTemplate method return void?

Reply:
Good point. It's not possible to do any verification as the method the library is exposing is void. I'll create an issue for library author to change the signature to return the id of deleted template